swift/integration-tests/osx-only/autobuilder/failure/diagnostics.expected

```diff
@@ -1,9 +1,8 @@
 {
   "helpLinks": [
-    "https://docs.github.com/en/enterprise-server/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/customizing-code-scanning",
-    "https://docs.github.com/en/enterprise-server/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-the-codeql-workflow-for-compiled-languages#adding-build-steps-for-a-compiled-language"
+    ""
```

Comment:
This doesn't look right. I'd expect there to be no help links in this case, rather than a single empty one?

I'm guessing this is an unintended artifact of using `absl::StrSplit` (which returns `{""}` on empty strings by design, [except in case of bugs](https://github.com/abseil/abseil-cpp/blob/master/absl/strings/str_split.h#L488-L492)).

Reply:
I'll fix this in a separate PR.

Reply:
Specifically https://github.com/github/codeql/pull/13224.